level-2/숫자의-표현.js

```diff
@@ -14,4 +14,22 @@ function solution(n) { //수학적 풀이는 별도로 하지 않았습니다.
         }
     }
     return answer;
+}
+
+//완벽한 정답이 아닙니다.
+//정답 2 - prove-ability
```

Comment:
해당 주석은 최상단에 있으므로 삭제해도 좋을 것 같습니다!
```suggestion
//정답 2 - prove-ability
```